services/movie.py

```diff
@@ -6,6 +7,7 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None
 ) -> QuerySet:
```

Comment:
You should specify inner type of `QuerySet`. Fix everywhere, please
```suggestion
) -> QuerySet[Movie]:
```